beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/AbstractBlockFactory.java

```diff
@@ -27,14 +25,4 @@ protected AbstractBlockFactory(
     this.spec = spec;
     this.operationSelector = operationSelector;
   }
-
-  @Override
```

Comment:
Do we actually need the AbstractBlockFactory anymore?

Reply:
good point not needed